readme-vars.yml

```diff
@@ -14,82 +14,97 @@ available_architectures:
 development_versions: true
 development_versions_items:
   - {tag: "latest", desc: "Stable Jellyfin releases"}
-  - {tag: "nightly", desc: "Unstable Jellyfin releases"}
+  - {tag: "nightly", desc: "Nightly Jellyfin releases"}
 # container parameters
 common_param_env_vars_enabled: true #PGID, PUID, etc
 param_container_name: "{{ project_name }}"
 param_usage_include_vols: true
 param_volumes:
   - {vol_path: "/config", vol_host_path: "/path/to/library", desc: "Jellyfin data storage location. *This can grow very large, 50gb+ is likely for a large collection.*"}
-  - {vol_path: "/data/tvshows", vol_host_path: "path/to/tvseries", desc: "Media goes here. Add as many as needed e.g. `/data/movies`, `/data/tv`, etc."}
+  - {vol_path: "/data/tvshows", vol_host_path: "/path/to/tvseries", desc: "Media goes here. Add as many as needed e.g. `/data/movies`, `/data/tv`, etc."}
   - {vol_path: "/data/movies", vol_host_path: "/path/to/movies", desc: "Media goes here. Add as many as needed e.g. `/data/movies`, `/data/tv`, etc."}
 param_usage_include_ports: true
 param_ports:
   - {external_port: "8096", internal_port: "8096", port_desc: "Http webUI."}
 param_usage_include_env: true
 param_env_vars:
-  - {env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London"}
+  - {env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use (e.g. Europe/London)."}
```

Comment:
The TZ var is templated, and this value is skipped.

Reply:
Gotcha, I copied from the master branch readme. I'll remove on both